bouncer/shared/broker_fee_collection.ts

```diff
@@ -173,19 +172,10 @@ async function testBrokerFees(asset: Asset, seed?: string): Promise<void> {
     [chain]: observeWithdrawalAddress,
   });
   console.log(`Submitted withdrawal for ${asset}`);
+
   const withdrawalRequestedEvent = await observeWithdrawalRequested.event;
+
   console.log(`Withdrawal requested, egressId: ${withdrawalRequestedEvent.data.egressId}`);
-  const BatchBroadcastRequestedEvent = await observeEvent(':BatchBroadcastRequested', {
```

Comment:
Was just testing an implementation detail that's not relevant, and was the cause of flakiness as it introduces a race condition.